cli/lib/tasks/unzip.js

```diff
@@ -73,12 +80,47 @@ const unzip = ({ zipFilePath, installDir, progress }) => {
           return extract(zipFilePath, opts, endFn)
         }
 
-        //# we attempt to first unzip with the native osx
-        //# ditto because its less likely to have problems
-        //# with corruption, symlinks, or icons causing failures
-        //# and can handle resource forks
-        //# http://automatica.com.au/2011/02/unzip-mac-os-x-zip-in-terminal/
+        const unzipWithUnzipTool = () => {
+          debug('unzipping via `unzip`')
+
+          const inflatingRe = /inflating:/
+
+          const sp = cp.spawn('unzip', ['-o', zipFilePath, '-d', installDir])
+
+          sp.on('error', unzipWithNode)
```

Comment:
oh, I see you fall to Node in that case

Reply:
ha `// f-it just unzip with node`

Reply:
i believe that is your comment :laughing: